src/ClusterBootstrap/az_tools.py

```diff
@@ -600,6 +602,19 @@ def get_disk_from_vm(vmname):
 
     return output.split("/")[-1].strip('\n')
 
+def AZvmsize2GPU(vmsize):
```

Comment:
Can we make this an Azure mapping file, e.g.
```
#sku gpu-type gpu-count
Standard_NC6s_v2 P100 1
...
```

Reply:
sure. should I put this mapping file to src/utils, in json format?